examples/tictactoe/.eslintrc.json

```diff
@@ -93,6 +94,11 @@
   },
   "ignorePatterns": ["**/*.umd.js", "**/generated/*"],
   "settings": {
+    "import/resolver": {
```

Comment:
why this rule?

Reply:
It was there to test some behaviour for eslint. removed it now